backend/apps/events/api/v1/routes.py

```diff
@@ -0,0 +1,11 @@
+from apps.events.api.v1.views import ListCreateEventAPIView, RetrieveUpdateDestroyEventView
+from django.urls import include, path
+from rest_framework import routers
+
+router = routers.DefaultRouter()
+router.register(r"", ListCreateEventAPIView, basename="events")
+
+urlpatterns = [
+    path("", include(router.urls)),
+    path(r"<int:pk>/", RetrieveUpdateDestroyEventView.as_view(), name="event-rud-view"),
```

Comment:
`name="event-rud-view"`
Not sure, but I think, this name is for the URL, not for view. If so, maybe we should rename it?

Reply:
This is indeed a name which is used in reversing, I used "event-rud-view", so that I would remember what to write in my tests. Can be anything, I guess =)